revised autocapture remote explanation (#498)
amplitude/amplitude-docs: content/collections/browser_sdk/en/browser-sdk-2.md

Beginning with version 2.10.0, the Amplitude Browser SDK supports remote configuration. By default, the SDK disables this feature.

> Autocapture supports remote configuration options for tracking default events, which will override your client-side configuration when this setting is enabled. You can find your remote configuration options in Data > Settings > Autocapture. 

Reviewer: ```suggestion
Autocapture supports remote configuration options for tracking default events. When you enable this setting, the remote configuration overrides your client-side configuration. Find the remote configuration options in *Data > Settings > Autocapture*. 
```
